src/app/modules/data-set/model/instance/instance.model.ts

```diff
@@ -23,9 +25,18 @@ export class Instance {
             this.setMetricFeatures(obj.metricFeatures);
             this.setType(obj.type);
             this.setAnnotationFromLoggedUser();
+            this.relatedInstances = this.setRelatedInstances(obj.relatedInstances);
```

Comment:
this.relatedInstances = obj.relatedInstances.map(ri => new RelatedInstance(ri));